content/docs/deploy/enterprise/changelog.mdx

```diff
@@ -7,6 +7,25 @@ toc_max_heading_level: 5
 
 # Changelog
 
+## 0.25.0
+
+### Breaking
+
+- In this release, we removed support for the `debug` option in the Enterprise.
```

Comment:
Is "the Enterprise" correct? Should this be "the Enterprise Console"?

Reply:
This was per @desimone 's edits, but I think Enterprise Console sounds fine.

Reply:
```suggestion
- In this release, we removed support for the `debug` option in the Enterprise Console.
```